homeassistant/components/notion/binary_sensor.py

```diff
@@ -28,17 +33,45 @@
     SENSOR_WINDOW_HINGED_VERTICAL,
 )
 
-BINARY_SENSOR_TYPES = {
-    SENSOR_BATTERY: ("Low Battery", "battery"),
-    SENSOR_DOOR: ("Door", DEVICE_CLASS_DOOR),
-    SENSOR_GARAGE_DOOR: ("Garage Door", "garage_door"),
-    SENSOR_LEAK: ("Leak Detector", DEVICE_CLASS_MOISTURE),
-    SENSOR_MISSING: ("Missing", DEVICE_CLASS_CONNECTIVITY),
-    SENSOR_SAFE: ("Safe", DEVICE_CLASS_DOOR),
-    SENSOR_SLIDING: ("Sliding Door/Window", DEVICE_CLASS_DOOR),
-    SENSOR_SMOKE_CO: ("Smoke/Carbon Monoxide Detector", DEVICE_CLASS_SMOKE),
-    SENSOR_WINDOW_HINGED_HORIZONTAL: ("Hinged Window", DEVICE_CLASS_WINDOW),
-    SENSOR_WINDOW_HINGED_VERTICAL: ("Hinged Window", DEVICE_CLASS_WINDOW),
+BINARY_SENSOR_DESCRIPTIONS: dict[str, BinarySensorEntityDescription] = {
+    SENSOR_BATTERY: BinarySensorEntityDescription(
+        key=SENSOR_BATTERY, name="Low Battery", device_class=DEVICE_CLASS_BATTERY
```

Comment:
I would recommend to add trailing commas. That way each key / value pair will be on a new line which makes it much easier to read.